From 54ac021920c8975d3fa6fc6ad145627a85f87a9b Mon Sep 17 00:00:00 2001
From: Luigi Toscano <luigi.toscano@tiscali.it>
Date: Sun, 21 Mar 2021 23:44:08 +0100
Subject: [PATCH 37/54] kundo2_aware_xgettext.sh: fix a gawk warning

The following warning is reported by gawk >= 5 but it really affects
also the previous versions:

warning: regexp escape sequence `\"' is not a known regexp operator
(cherry picked from commit 15b70303e2b433e6fbb10e90924d7dbd801cf960)
---
 kundo2_aware_xgettext.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kundo2_aware_xgettext.sh b/kundo2_aware_xgettext.sh
index b5921d93e2c..0249b68b03d 100644
--- a/kundo2_aware_xgettext.sh
+++ b/kundo2_aware_xgettext.sh
@@ -38,7 +38,7 @@ function add_ctxt_qtundo() {
     mv "${POT_PART_QUNDOFORMAT}" "${POT_PART_QUNDOFORMAT2}"
     cat "${POT_PART_QUNDOFORMAT2}" | awk '
         /^msgid "/ {
-            if (lastLine !~ /^\"/ && lastLine !~ /^msgctxt/ && lastLine != "#, fuzzy") {
+            if (lastLine !~ /^"/ && lastLine !~ /^msgctxt/ && lastLine != "#, fuzzy") {
                 print "msgctxt \"(qtundo-format)\""
             }
         }
-- 
2.31.1

